demo/run_demo.py

```diff
@@ -34,6 +34,13 @@ def supply_chain():
   subprocess.call(write_code_cmd.split())
   copyfile("foo.py", "../functionary_carl/foo.py")
```

Comment:
As we discussed, I have kept the demo as it was earlier. Just changed a few minor things like the link names ([step-name].[keyid].link)